apps/admin-x-activitypub/src/components/feed/FeedItem.tsx

```
@@ -262,18 +271,25 @@
-                                            size='md'
-                                        />}
+                                        {(object.type === 'Article') ? <div className='rounded-md border border-grey-150 transition-colors hover:bg-grey-75'>
+                                            {renderFeedAttachment(object, layout)}
+                                            <div className='p-4'>
+                                                <Heading className='mb-1 text-pretty leading-tight' level={5} data-test-activity-heading>{object.name}</Heading>
+                                                <div className='line-clamp-3 leading-tight'>{object.preview?.content}</div>
+                                            </div>
+                                        </div> :
+                                            <div className='relative'>
+                                                <div
+                                                    dangerouslySetInnerHTML={({__html: object.content ?? ''})}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)